SU-SWS/stanford_profile
# CSD-9 Fix up for su-hero-height styling

# READY FOR REVIEW

# Summary
- `su-hero-height` styling was not applying correctly for anonymous users due to an additional div (that is normally not present when user is logged in)

# Needed By (Date)
- asap

# Urgency
- medium

# Steps to Test

1. Pull to local
2. Add `su-hero-height` as an Extra class for homepage filter banner.
3. Clear cache and log out.
4. View homepage and select filters.

Outcome: Banner image should span. No grey bar should appear as the banner flexes

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
